redisearch/aggregate.go

```diff
@@ -187,6 +187,16 @@ func (a *AggregateQuery) Limit(offset int, num int) *AggregateQuery {
 	return a
 }
 
+//Sets the limit for the initial pool of results from the query.
+func (a *AggregateQuery) Load( Properties []string) *AggregateQuery {
+	nproperties := len(Properties)
+	if nproperties > 0 {
+		a.AggregatePlan = a.AggregatePlan.Add("LOAD", nproperties)
+		a.AggregatePlan = a.AggregatePlan.AddFlat( Properties )
```

Comment:
I missed where you add the `@` to the fields names

Reply:
totally forgot about it Meir. Already corrected.